e2e/tests/images_update.yaml

```diff
@@ -0,0 +1,8 @@
+kind: BaseImage
+name: nodejs6-base
+dockerUrl: vmware/dispatch-nodejs6-base:0.0.1-dev1
+language: python3
+tags:
+  - key: role
+    value: test
+---
```

Comment:
is this supposed to be here?

Reply:
I had added a test for images as well, but that was failing because the PR to support image updates hadn't been pushed yet. I'm currently adding that test back in and should have an updated version of this PR soon.